docs/source/DeveloperGuide/AddingNewComponents/DevelopingDatabases/index.rst

```diff
@@ -50,34 +50,29 @@ The *addStuff* method simply inserts into the created table the argument value.
 Configure the database access
 ==============================
 
-The last step is to configure the database credentials for DIRAC to be able to connect. In our previous example the CS path was */Systems/Test/<instance name>/Databases/AtomDB*. That section should contain::
+The last step is to configure the database credentials for DIRAC to be able to connect. In our previous example the CS path was */Systems/<instance name>/Databases/AtomDB*. That section should contain::
 
   Systems
   {
-    Test
+    <instance name>
```

Comment:
This is rather the system name, which is supposed to be a Test in this context ?